### PR TITLE
Update forge version

### PR DESCRIPTION
**Pull request by modmuss50**
Dont merge just yet.

This PR updates the forge version to the latest. I plan to fix some of the other issues over the next few days.

## Known issues:

- ~~I disabled the re-mapping event for now, I will add that back in later~~

- Some items arent rendering correctly

The examples on the new FTB map seem to work fine apart from that NO testing has been done.

Note: This is my first time using Scala so please point out where I went wrong.

### CODE REVIEW

**Comment by modmuss50** [comment]
Ok, I think this is all I can do. Apart from most of the items not rendering everything seems to work ok. If anyone has any input that could help me fix the items please let me know.

File: src/main/scala/li/cil/oc/common/init/Items.scala [hunk 1]
```diff
@@ -52,26 +52,26 @@ object Items extends ItemAPI {
...
 
-  def registerBlock[T <: Block](instance: T, id: String): T = {
+  def registerBlock(instance: Block, id: String): Block = {
     if (!descriptors.contains(id)) {
       instance match {
         case simple: SimpleBlock =>
           instance.setUnlocalizedName("oc." + id)
           instance.setRegistryName(id)
-          GameRegistry.register(instance)
+          GameData.register_impl(instance)
```
Comment (thiakil) [inline src/main/scala/li/cil/oc/common/init/Items.scala:61]: in 1.12 you need to use the event! `register_impl` is an internal use method as far as I know. The models should also usually be registered in the event, after the block/item itself has registered

**Comment by modmuss50** [comment]
Humm,I think I tried that, it's a bit too much work for me to move everything to events, and after speaking to @Vexatos he agreed to stick with the GameData method. (I use it for all my other mods just fine) I could try registering the models in the event though.

**Comment by fnuecke** [comment]
Thanks for getting it this far! I'll see how much of what remains I can get done today.

**Pull request merged**